src/Sentry.Serilog/SentrySinkExtensions.cs

```diff
@@ -189,7 +192,7 @@ public static class SentrySinkExtensions
         bool? initializeSdk = null,
         Dictionary<string, string>? defaultTags = null)
     {
-        if (!string.IsNullOrWhiteSpace(dsn))
+        if (dsn != null)
```

Comment:
prefer pattern matching over operator

```suggestion
        if (dsn is not null)
```